src/utils.js

```diff
@@ -75,6 +75,11 @@ const joinOrLeaveToEvent = async (textEventButton, userId, eventDetail) => {
 };
 
 const getTextButton = (userId, eventDetail) => {
+  if (typeof eventDetail === "string") {
+    const events = getEventsFromLocalStorage().events;
+    eventDetail = events.find((event) => event.id === eventDetail);
+  }
```

Comment:
this is not a valid solution, we always should receive `eventDetail` as an object, we need to analyze in which case we are passing `eventDetail` as string, because we can't call the `getTextButton` if `eventDetail` is a string